mongo/assets/configuration/spec.yaml

```diff
@@ -61,7 +61,7 @@ files:
         properties: []
     - name: connection_scheme
       description: |
-        A prefix indicating the connection string format e.g. mongodb+srv.
+        A prefix indicating the connection string format.
```

Comment:
The only valid connection_schemes are `mongodb` and `mongodb+srv`.
Let's remove that configuration option completely

Reply:
Deprecating the config option then. `changelog/changed` seems too much, so i'm using `changelog/fixed`